lib/cards/KCard.vue

```diff
@@ -177,34 +200,32 @@
         },
       },
       /**
-       * Sets card title if provided. The title should be
-       * unique within a grid and descriptive to aid screenreader
-       * navigation.
+       * Card title
        */
```

Comment:
The check is in `KCard`'s `mounted`, looks good or do you think it needs to be adjusted somehow?

Reply:
You're right, the check in the mounted hook does ensure that either the title slot or prop is present. I was just thinking that since the title prop defaults to null, an explicit check for null or undefined might be worth considering, just to avoid any edge cases where the prop is technically passed but empty. Otherwise, it looks good!

Reply:
@AllanOXDi this doesn't seem blocking and I've just addressed remaining feedback so we can merge before the next release - still you're welcome to let me know and I will take care of it in one of the upcoming PRs.